projects/angular-showcase/src/app/public/examples/autocomplete-showcase/autocomplete-showcase.component.ts

```diff
@@ -3,17 +3,19 @@ import { FormControl } from '@angular/forms';
 import { Subject } from 'rxjs';
 import { debounceTime, distinctUntilChanged, switchMap } from 'rxjs/operators';
 
+const MAX_OPTIONS_LIST_LENGTH = 5;
```

Comment:
I'd refactor this just to a readonly property, since I see no need for the const here:
`readonly maxOptionsListLength = 5;`